internal/providers/selectors/selector_entity.go

```diff
@@ -0,0 +1,127 @@
+//
+// Copyright 2024 Stacklok, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package selectors
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/protobuf/proto"
+
+	internalpb "github.com/stacklok/minder/internal/proto"
+	minderv1 "github.com/stacklok/minder/pkg/api/protobuf/go/minder/v1"
+	provifv1 "github.com/stacklok/minder/pkg/providers/v1"
+)
+
+// entityInfoConverter is an interface for converting an entity from an EntityInfoWrapper to a SelectorEntity
+type entityInfoConverter interface {
```

Comment:
just noticed the interface is misnamed, it's no longer tied to `EntityInfoWrapper`, but just a pair an entity as a `proto.message` and entity type.